config/motd.txt

```diff
@@ -1,5 +1,14 @@
-<!-- Most valid html will work in here, excluding images. -->
+<h1 style="font-family: Verdana; margin: 0px; padding: 0px; text-align: center;">Welcome to Yogstation 13!</h1>
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">We have a support ticket system at Yogstation. If you have an issue, please file a ticket by pressing <b>F1</b> or going to the <b>admin tab</b> and clicking <b>Ahelp</b>. An admin will help you as soon as possible.</span>
 
-<h1>Welcome to Space Station 13!</h1>
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">We are constantly seeking dedicated new <span style="font-weight: bold; color: #dd4400;">admins</span>, <span style="font-weight: bold; color: #dd4400;">coders</span>, <span style="font-weight: bold; color: #dd4400;">spriters</span> and <span style="font-weight: bold; color: #dd4400;">mappers</span> to improve our server. Without them, you wouldn't have a server to play on. If you feel you may be a good addition to our team, please apply:</span>
+<span style="font-family: Verdana; margin: 0px; padding: 0px;">
+ ¤ <a href="http://forums.yogstation.net/index.php?forms/admin-application-form.4/respond">Admin Application</a>
+ ¤ <a href="http://forums.yogstation.net/index.php?forms/coder-application-form.12/respond">Coder / Spriter / Mapper Application</a>
```

Comment:
This is outdated and should probably be rewritten

Reply:
meh, I just ported old one over, someone can rewrite it if they feel like it